truss/tests/test_config.py

```diff
@@ -206,47 +206,49 @@ def test_non_default_train():
     assert new_config == config.to_dict(verbose=False)
 
 
-def test_null_hf_cache_key():
-    config_yaml_dict = {"hf_cache": None}
+def test_null_model_cache_key():
+    config_yaml_dict = {"model_cache": None}
```

Comment:
In case I missed this, could we have a case where there's a yaml file with the key "hf_cache" and check that it parses out correctly?